collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -204,11 +204,38 @@ bool NetworkStatusNotifier::UpdateAllConnsAndEndpoints() {
   return true;
 }
 
+bool NetworkStatusNotifier::ExternalContainerEndpointEquality::operator()(const ContainerEndpoint& lhs, const ContainerEndpoint& rhs) const {
+  if (bool(lhs.originator()) != bool(lhs.originator())) {
+    return false;
+  }
+
+  if (lhs.originator()) {
+    auto& lhs_oritinator = *lhs.originator();
+    auto& rhs_oritinator = *rhs.originator();
```

Comment:
```suggestion
    auto& rhs_originator = *rhs.originator();
```
Fix other typo as well.